src/librustdoc/html/static/main.js

```diff
@@ -696,39 +703,52 @@
             var search_input = document.getElementsByClassName('search-input')[0];
             search_input.onkeydown = null;
```

Comment:
Is that line still required?

Reply:
Nope. Removing it. Thanks for spotting it!